### ISSUE TITLE
🐛 Biome encountered an unexpected error

### ISSUE THREAD

**Issue by sharathvgts** [biomejs/biome#8054]
### Environment information

```block
CLI:
  Version:                      2.3.4
  Color support:                true

Platform:
  CPU Architecture:             aarch64
  OS:                           macos

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  BIOME_THREADS:                unset
  NO_COLOR:                     unset
  TERM:                         xterm-256color
  JS_RUNTIME_VERSION:           unset
  JS_RUNTIME_NAME:              unset
  NODE_PACKAGE_MANAGER:         unset

Biome Configuration:
  Status:                       Loaded successfully
  Path:                         biome.json
  Formatter enabled:            true
  Linter enabled:               true
  Assist enabled:               true
  VCS enabled:                  true

Workspace:
  Open Documents:               0
```

### What happened?

Biome encountered an unexpected error

This is a bug in Biome, not an error in your code, and we would appreciate it if you could report it to https://github.com/biomejs/biome/issues/ along with the following information to help us fixing the issue:

Source Location: crates/biome_service/src/settings.rs:78:14
Thread Name: tokio-runtime-worker
Message: You must have at least one workspace.

### Expected result

It should work

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by sharathvgts** [comment]
Fixed when I updated the biome version.
